src/inputs/Table/Table.js

```diff
@@ -517,6 +524,11 @@ Table.propTypes = {
    * If not defined, this button will not be rendered.
    */
   onDeleteRow: PropTypes.func,
+  /*
+   * Callback function that will be called when user clicks on Revert button.
```

Comment:
Is *Revert* a component name ? In this case, the higher case would be justified but in other case, shouldn't we prefer full low cases to indicate the main purpose of a component ?

Reply:
The full component name is `RevertDataButton`, I decided to shorten it to `Revert` to be more concise but we can use lower case to point out the function as in other callback descriptions :wink: